grammars/less.cson

```diff
@@ -456,3 +453,51 @@
   'less_builtin_functions':
     'match': '\\b(abs|acos|alpha|argb|asin|atan|average|blue|calc|ceil|color|contrast|convert|convert|cos|darken|data-uri|desaturate|difference|e|escape|exclusion|extract|fade|fadein|fadeout|floor|format|green|greyscale|hardlight|hsl|hsla|hsv|hsva|hsvhue|hsvsaturation|hsvvalue|hue|length|lighten|lightness|luma|max|min|mix|mod|multiply|negation|overlay|percentage|pi|pow|red|replace|round|saturate|saturation|screen|sin|softlight|spin|sqrt|tan|unit)\\b'
     'name': 'support.function.any-method.builtin.less'
+  'mixins':
+    'name': 'meta.mixin.less'
+    'begin': '(\\.[_a-zA-Z][a-zA-Z0-9_-]*)(?=\\()'
+    'beginCaptures':
+      '0':
+        'name': 'meta.definition.mixin.less entity.name.mixin.less'
+    'end': '(?=$|^|;)|(?<=\\})'
+    'patterns': [
+      {
+        'include': '#mixin-parameters'
+      }
+    ]
+  'mixin-parameters':
+    'name': 'meta.parameters.less'
+    'begin': '\\('
+    'beginCaptures':
+      '0':
+        'name': 'punctuation.definition.parameters.begin.bracket.round.less'
+    'end': '\\)'
+    'endCaptures':
+      '0':
+        'name': 'punctuation.definition.parameters.end.bracket.round.less'
+    'patterns': [
+      {
+        'include': '#parameter'
+      }
+    ]
+  'parameter':
+    'begin': '(@[a-zA-Z][a-zA-Z0-9-]*)'
+    'beginCaptures':
+      '0':
+        'name': 'variable.parameter.less'
+    'end': '(?:([,;])|(?=\\)))'
+    'endCaptures':
+      '0':
+        'name': 'punctuation.separator.parameter.less'
+    'patterns': [
+      {
+        'match': ':'
+        'name': 'punctuation.separator.key-value.less'
```

Comment:
@50Wliu Should this be `punctuation.separator.key-value.css`? Or is it incorrectly set here: https://github.com/atom/language-less/pull/82/files#diff-14c55924b6cf1fe75bf40519b9cddaccR317?

Reply:
I think this can be kept as `.less`.

Reply:
Should I change the other occurrence then?

Reply:
The other one can be kept as `.css` because that one is the same in CSS, whereas for this one there is no function equivalent in CSS.

It's a bit confusing and is pretty ad-hoc.  Hopefully in the future the endings can be standardized.